src/rv2/core/labels.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC, abstractmethod
+
+
+class Labels(ABC):
+    '''A set of spatially referenced labels.
+
+    A set of labels associated with their location predicted by a model or
+    provided by human labelers for the sake of training. An example of an label
+    is a bounding box in the case of object detection or a raster in case of
+    segmentation.
+    '''
```

Comment:
Just to be clear: for classification, the classes are still spatially referenced, by tying the chip extent to the classification(s)?

Reply:
Yes. I'll clarify this in the comment.